go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/canonical/jimm/v3
 
-go 1.22.5
+go 1.23
```

Comment:
Why not `go 1.23.2` and remove the `toolchain` statement?

Reply:
I've removed it now aha, it was autoadded